shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/modules/login/LoginViewModel.kt

```diff
@@ -23,18 +25,39 @@ class LoginViewModel(private val logInUseCase: LogInUseCase): BaseViewModel() {
 
     val viewState: StateFlow<LoginViewState> = mutableViewState
 
-    fun logIn(email: String, password: String) {
-        mutableViewState.update {
-            LoginViewState(isLoading = true)
+    fun login(email: String, password: String) {
+        if (!validInput(email, password)) {
+            return
         }
```

Comment:
You can apply this one-line style and remove the brackets also 😁
```suggestion
        if (!validInput(email, password)) return
```